runtimes/pythonrt/local_runner.go

```diff
@@ -135,7 +135,12 @@ func (r *LocalPython) Start(pyExe string, tarData []byte, env map[string]string,
 		return fmt.Errorf("copy runner code - %w", err)
 	}
 
-	r.id = uuid.NewString()
+	id, err := typeid.WithPrefix("runner")
```

Comment:
Do we really need it?

Reply:
Yes, there's a check with `buf` that identifiers are `typeid`. Took me a while to catch that (with help of @itayd )